pallets/nfts/src/tests.rs

```diff
@@ -273,35 +274,35 @@ fn lifecycle_should_work() {
 			account(10),
 			default_item_config()
 		));
-		assert_eq!(Balances::reserved_balance(&account(1)), 6);
+		assert_eq!(Balances::reserved_balance(&account(1)), 6 + 1); // 1 - balance deposit
```

Comment:
Can we use the config `T::BalanceDeposit`, see increment_account_balance_works comment.

And everywhere else in the tests. Then we also don't need the comment as it explains itself.